assets/src/domain/eventEditor/services/apollo/queries/datetimes/useFetchDatetimes.ts

```diff
@@ -10,28 +11,32 @@ const useFetchDatetimes = (): FetchEntitiesResult<DatetimesList> => {
 	const { setIsLoading, setIsLoaded, setIsError } = useStatus();
 	const { query, ...options } = useDatetimeQueryOptions();
 
-	const toaster = useToaster();
-	const loadingToastKey = useRef(toaster.generateKey());
+	const toaster = useSystemNotifications();
+	const toastId = useRef(null);
 
-	const dismissLoading = (): void => toaster.dismiss(loadingToastKey.current);
+	const dismissLoading = (): void => toaster.dismiss(toastId.current);
 
 	const { data, error, loading } = useQuery<DatetimesList>(query, {
 		...options,
 		onCompleted: (): void => {
 			setIsLoaded(TypeName.datetimes, true);
+			setIsLoading(TypeName.datetimes, false);
 			dismissLoading();
-			toaster.success({ message: 'datetimes initialized' });
+			toastId.current = toaster.success({ message: 'datetimes initialized' });
```

Comment:
i just realized these messages are not i18n  🙁

Reply:
Yes, if toaster renders its own react instance, it won't be `i18` ready

Reply:
done